### ISSUE TITLE
🐞 Bug: broken simulator UI

### ISSUE THREAD

**Issue by pulkit-30** [CircuitVerse/CircuitVerse#4290]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

UI for the simulator seems broken.
- showing mobile small screen view on big screen size.

### Steps To Reproduce

1. go to /simulator
2. see the UI

### Expected Behavior

Ui should be consistent

### Screenshots

https://github.com/CircuitVerse/CircuitVerse/assets/76155456/e2c2dc63-301f-4611-9468-192bc016aef1

### Device Information [optional]

```markdown
- OS: Mac
- Browser: firefox
- version: latest
```

### Additional context

Need to fix this asap.

### Are you working on this issue?

No

**Comment by pulkit-30** [comment]
@tanmoysrt @Prerna-0202 @vedant-jain03 can you please confirm this issue?

**Comment by niladrix719** [comment]
if valid i would like to work on this

**Comment by tanmoysrt** [comment]
I noticed this 
1. When we have the `responsive icon` enabled in devtool and resizing size, it's working perfectly
2. but when we disable that option, it's acting as the video. Maybe in that case, the resize event listener doesn't work

**Comment by pulkit-30** [comment]
I am not sure about resize issue, @niladrix719 feel free to open PR for this.

**Comment by niladrix719** [comment]
@pulkit-30  i can't reproduce it, this might not be an error actually, may be it was some issue with the browser in the video

**Comment by fillingtothemomo** [comment]
@pulkit-30 I would like to take up this issue, please assign it to me.

**Comment by tanmoysrt** [comment]
> @pulkit-30 I would like to take up this issue, please assign it to me.

It's seem to be a issue on a specific version of browser or for any installed extensions.
If you are able to reproduce the same issue then you can investigate that

**Comment by fillingtothemomo** [comment]
https://github.com/CircuitVerse/CircuitVerse/assets/122573982/aacab26c-c1df-402b-9424-df77116641be
when I change the screen size the different dropdowns get mushed together but the navbar remains the same when developer tools are exited

**Comment by tanmoysrt** [comment]
> https://github.com/CircuitVerse/CircuitVerse/assets/122573982/aacab26c-c1df-402b-9424-df77116641be when I change the screen size the different dropdowns get mushed together but the navbar remains the same when developer tools are exited

Whenever you are working with responsive stuff, enable `device toolbar`
![image](https://github.com/CircuitVerse/CircuitVerse/assets/57363826/0a50a033-387f-499d-8be1-71baef56049e)
The laptop like icon.

With this options if you change the screen size, you are squeezing it, the window heightt and width does not modify.
But if you enable that, the window height and width changes accordingly and js css can works with that hwight.

You can tryout in any screen resolution testing website. for example - https://mdigi.tools/screen-resolution/#google_vignette

**Comment by fillingtothemomo** [comment]
![image](https://github.com/CircuitVerse/CircuitVerse/assets/122573982/c9b3bd93-29d4-46c1-bbf5-bd6342ef193b)
Thanks for the tip , the ui is still a bit mushed while resizing the screen

**Comment by tanmoysrt** [comment]
> ![image](https://private-user-images.githubusercontent.com/122573982/292881522-c9b3bd93-29d4-46c1-bbf5-bd6342ef193b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDM2MDE1NzIsIm5iZiI6MTcwMzYwMTI3MiwicGF0aCI6Ii8xMjI1NzM5ODIvMjkyODgxNTIyLWM5YjNiZDkzLTI5ZDQtNDZjMS1iYmY1LWJkNjM0MmVmMTkzYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIyNlQxNDM0MzJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00NWFlZTU5OTIwZTg1ZmQ4N2Q5ZGUzZjgzZmIyOTVmN2U0OWM5ZWUwNTc4MTdlNGFmNmQ3MDRmMTY1YWZjNzVlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Dc8aKAnnGVv59MT5Nv2ptS3vMqvH-xWhMjINY76JgZI) Thanks for the tip , the ui is still a bit mushed while resizing the screen ![image](https://private-user-images.githubusercontent.com/122573982/292881842-79515924-a9ad-4ce2-9ce5-1d8c6870d774.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDM2MDE1NzIsIm5iZiI6MTcwMzYwMTI3MiwicGF0aCI6Ii8xMjI1NzM5ODIvMjkyODgxODQyLTc5NTE1OTI0LWE5YWQtNGNlMi05Y2U1LTFkOGM2ODcwZDc3NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIyNlQxNDM0MzJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMjc2YjI3MTZkNTJhNDgxMGVkMzFhOWNiOTFkNzhmMjI3ZGFmNmIwNTFlNDIxNTVhMTkxOGYxOGQ5MmQzNmJiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.dO0D-70DrXan7gdcTidrKu-zc87yxNa07Vj5UiJtyFY) Also the profile dropdown icon in the navbar is cutoff a bit

Yes the username overflow issue may be fixed , as far I remember. The production website hasn't latest changes.

**Comment by Pulkit0729** [comment]
This issue is still there, it is because is our media queries we are using `max-device-width`, instead of` max-width`, the issue is clearly explained here https://stackoverflow.com/a/68048769

@tanmoysrt, the solutiion would be to change the respective media queries, if you agree, i can raise a pr on that

**Comment by niladrix719** [comment]
@Pulkit0729 but if changed to max-width will that not effect the computer screens when squeezing? can you tell me how did you reproduce the error.

**Comment by Pulkit0729** [comment]
@niladrix719 Here are the videos for before and after

### Before max-device-width

Uploading screen-capture (1) (online-video-cutter.com).mp4…

### After max-width

Uploading after.mp4…

I did not understand by what you mean by computer screen while squeezing?
